client/src/App.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import './App.css';
+import graphql from 'babel-plugin-relay/macro';
+import {
+  RelayEnvironmentProvider,
+  loadQuery,
+  usePreloadedQuery,
+} from 'react-relay/hooks';
+import RelayEnvironment from './RelayEnvironment';
+
+const { Suspense } = React;
+
+// Define a query
+const AppAllTodosQuery = graphql`
+query AppQuery {
+  allTodos {
+    nodes {
+      id,
+      task,
+      completed,
+      dateCreated,
+      dateUpdated
+    }
+  }
+}
```

Comment:
At first I tried copy/pasting the query I made in graphiql, but I got errors and ended up tweaking to this. Should copy/pasted graphiql queries work, or is it normal to use a slightly different format for Relay?

Reply:
Other than Relay having requirements when it comes to query/mutation/fragment naming conventions, they should work the same. What errors did you have?

Reply:
I must have tried it before troubleshooting babel because it's working now. Thanks!